.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Please follow the guide below
+
+- You will be asked some questions, please read them **carefully** and answer honestly
+- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
+- Use *Preview* tab to see how your *pull request* will actually look like
+
+- [ ] [Searched](https://github.com/FelisCatus/SwitchyOmega/search?q=is%3Apr&type=Issues) for similar pull requests

```

Comment:
Is this link correct?

Reply:
The search result from `https://github.com/FelisCatus/SwitchyOmega/search?q=is%3Apr&type=Issues` could show summary of the content rather than `https://github.com/FelisCatus/SwitchyOmega/pulls?utf8=%E2%9C%93&q=is%3Apr`